src/pymor/parameters/base.py

```diff
@@ -245,32 +264,63 @@ def __hash__(self):
         return hash(tuple(self.items()))
 
 
-class Mu(SortedFrozenDict):
+class Mu(FrozenDict):
```

Comment:
I think it's weird that we internally give up the sorting, but pretend that in the representation.

Reply:
I'm not quite sure what you mean by that. `Mu`s *are* sorted (see the for loop).

Reply:
Then it's not immediately obvious to me why there's sorting in `__repr__`

Reply:
Because `__repr__` uses `_raw_values` which are not sorted for efficiency reasons.